libs/api/domains/user-profile/src/lib/userProfile.resolver.ts

```diff
@@ -72,4 +75,20 @@ export class UserProfileResolver {
   ): Promise<ConfirmResponse | null> {
     return await this.userUserProfileService.confirmEmail(input, user)
   }
+
+  @Mutation(() => UserDeviceToken)
+  async addDeviceToken(
+    @Args('input') input: UserDeviceTokenInput,
+    @CurrentUser() user: User,
+  ): Promise<UserDeviceToken> {
+    return await this.userUserProfileService.addDeviceToken(input, user)
+  }
+
+  @Mutation(() => DeleteTokenResponse)
+  async deleteDeviceToken(
```

Comment:
Both mutation not following naming conventions for island.is ... missing the domain name in the entities. This might override other existing resolvers.

ref: https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-naming-conventions-into-shared-api

Reply:
addressed in latest commit